index.bs

```diff
@@ -573,7 +573,7 @@ dictionary CookieListItem {
   CookieSameSite sameSite;
 };
 
-typedef sequence<CookieListItem> CookieList;
+typedef FrozenArray<CookieListItem> CookieList;
```

Comment:
I don't think this is correct. All the usage of CookieList outside of CookieChangeEvent should probably still be a sequence rather than a FrozenArray (i.e. getAll returns a `Promise<sequence<CookieListItem>>`, CookieChangeEventInit should probably contain `sequence<CookieListItem>` members).

Reply:
Ah I see what you mean, thanks for pointing this out! I'll keep this one `sequence<CookieListItem>` and change the CookieChangeEvent to be `FrozenArray<CookieListItem>`. Let me know if that looks ok.